### ISSUE TITLE
User executing Pull Request

### ISSUE THREAD

**Issue by dkurniawan1990** [jan-molak/jenkins-build-monitor-plugin#253]
I think it would be great if there's an option to show the name of the user who executed the pull request in the dashboard so that if it fails we can directly know the on responsible for the build.

**Comment by jan-molak** [comment]
Interesting, so something along the lines of "Building pull request from @dkurniawan1990"?

**Comment by dkurniawan1990** [comment]
yes, that would be great, so we know who is in charge of the last pull request right from the dashboard.

**Comment by jan-molak** [comment]
OK, I'll add this to the backlog, PRs welcome!

**Comment by deiga** [comment]
When this is added, please add an configuration option for it. Our build monitor is pretty cluttered already and more text would be horrible

**Comment by yandemontreal** [comment]
Could this be generalized (and configurable) to show the pull request title, as well as source and target branches?
